### PR TITLE
Use babel and babel-loader instead of node-jsx and jsx-loader

### PR DESCRIPTION
**Pull request by mridgway**
Allows ES6 in applications.

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by Vijar** [comment]
:ship:

File: app/templates/_package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "lint": "grunt jshint"
   },
   "dependencies": {
+    "babel": "^4.6.3",
```
Comment (akshayp) [inline app/templates/_package.json]: node-jsx and jsx-loader should probably be removed from the deps

Comment (mridgway) [inline app/templates/_package.json]: Good call. Don't know how I missed that.

Comment (mridgway) [inline app/templates/_package.json]: Fixed.

**Comment by caridy** [comment]
+1

I guess we are ok with monkey patch require for the server side. /cc @ericf

**Pull request merged**